rosidl_generator_c/CMakeLists.txt

```diff
@@ -13,8 +13,10 @@ endif()
 
 find_package(ament_cmake_python REQUIRED)
 find_package(ament_cmake_ros REQUIRED)
+find_package(rcutils REQUIRED)
 
 ament_export_dependencies(rosidl_cmake)
+ament_export_dependencies(rcutils)
```

Comment:
Since this is adding a dependency to the generated code, I don't think `rcutils` needs to be exported. Instead it needs to be added as a dependency on the generated target.

https://github.com/ros2/rosidl/blob/dceaeb91f72a72e3867557690e2cc074e349230c/rosidl_generator_c/cmake/rosidl_generator_c_generate_interfaces.cmake#L131

Reply:
Addressed